menas/ui/components/schema/schemaDetail.controller.js

```diff
@@ -334,10 +335,19 @@ sap.ui.define([
       }
     },
 
+    httpUrlRx: new RegExp('^(https?:\\/\\/)?'+ // protocol
```

Comment:
Tried a few usual URLs. The only that didn't work is `::1`

Reply:
That seems bearable.

Reply:
OK, I have updated the IPv6 part a bit to allow shorted forms and thus also ::1.